managers/asset-operations-manager.js

```diff
@@ -829,29 +836,41 @@ class AssetOperationsManager {
                 blockchain,
             );
 
-            tokenAmountInWei = await this._getUpdateBidSuggestion(
-                UAL,
+            const firstAssertionId = await this.blockchainService.getAssertionIdByIndex(
+                tokenId,
+                0,
                 blockchain,
+            );
+
+            const keyword = ethers.solidityPacked(
+                ['address', 'bytes32'],
+                [contract, firstAssertionId],
+            );
+
+            const agreementId = ethers.sha256(
+                ethers.solidityPacked(
+                    ['address', 'uint256', 'bytes'],
+                    [contract, tokenId, keyword],
+                ),
+            );
+            const agreementData = await this.blockchainService.getAgreementData(
+                agreementId,
+                blockchain,
+            );
+
+            tokenAmountInWei = await this.nodeApiService.getBidSuggestion(
                 endpoint,
                 port,
                 authToken,
-                latestFinalizedState,
+                blockchain.name.startsWith('otp') ? 'otp' : blockchain.name,
+                epochsNumber - agreementData.epochsNumber,
```

Comment:
This is not correct, epochsNumber is amount of additional epochs, not new value

Reply:
If its the amount of additional epochs then it should be correct. You only need to get bid suggestion for new epochs